sdk/sdk-skeleton.go

```diff
@@ -0,0 +1,126 @@
+import "time"
+
+type Asset struct{}
+
+//// Methods directly on the SDK
+
+func NewChannel() {
+
+}
+
+func NewChannelListenerService() {
+
+}
+
+func GetChannel() {
+
+}
+
+//// Channel State Machine
+
+type ChannelStateMachine struct {
+	ChannelID              string
```

Comment:
Let's drop the channel ID. I don't see how that would be used internally at the moment.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/25/commits/ce73653f4ebdd5884d37aa91946d147fe348610a